src/sql/src/plan/with_options.rs

```diff
@@ -140,10 +149,13 @@ impl<V: TryFromValue<Value>> ImpliedValue for Vec<V> {
     }
 }
 
-impl<V: TryFromValue<Value>, T: AstInfo> TryFromValue<WithOptionValue<T>> for V {
+impl<V: TryFromValue<Value> + StrConv, T: AstInfo + std::fmt::Debug>
+    TryFromValue<WithOptionValue<T>> for V
+{
     fn try_from_value(v: WithOptionValue<T>) -> Result<Self, anyhow::Error> {
         match v {
             WithOptionValue::Value(v) => V::try_from_value(v),
+            WithOptionValue::Ident(i) => Ok(V::strconv(i.as_str())?),
```

Comment:
Still digging in, but it seems like we can avoid this and the `StrConv` trait and associated macros with something like:

```
WithOptionValue::Ident(i) => V::try_from_value(Value::string(i.as_str())),
```

Reply:
Definitely--when I woke up this morning, I had the same idea.